rdkcentral/Dobby
# RDKTV-13083 Get backup config if current one is corrupted

### Description
In case problems with parsing config try to get backup and run container from it.

### Test Procedure
Run any container once (to make sure backup is created). Stop container and then remove some content from config.json so it is no longer valid (i.e. any brace symbol). Try to run container again, with this fix everything should work fine

### Type of Change

### Requires Bitbake Recipe changes?
